docs/resources/projects/settings/general-settings.md

```diff
@@ -0,0 +1,195 @@
+---
+slug: general-settings
+title: General Settings
+tags: [project, assets, splash screen, launcher icon, adaptive icon]
```

Comment:
Tags should start with capital letters. And they should be relevant to multiple pages. Keywords can be any but tags need to be few.

Reply:
Only Project and Assets should be there in tags.

Reply:
Updated!